config.private.json

```diff
@@ -1,3 +1,3 @@
 {
-  "api": "https://wwwint.ebi.ac.uk/metagenomics/api/v1/"
```

Comment:
We should indeed change this default at some point... but right now I think this will break the deployment to wwwint... Can you rollback this change for now please?

Reply:
@SandyRogers I have reverted this back to it's original form